packages/regl-worldview/src/commands/GLText.js

```diff
@@ -211,6 +211,7 @@ const frag = `
   uniform float cutoff;
   uniform bool scaleInvariant;
   uniform float scaleInvariantSize;
+  uniform bool forHitmap;
```

Comment:
We use `isHitmap` in other files, would be nice to keep consistent

Reply:
Yes, I agree. We should keep the code as consistent as possible. I already updated my code to use isHitmap. Thanks!